en/docs/security-announcements/security-advisories/2025/WSO2-2025-4486.md

```
@@ -0,0 +1,90 @@
+
+# Security Advisory WSO2-2025-4486/CVE-2025-10853
+
+<p class="doc-info">Published: 2025-11-05</p> 
+<p class="doc-info">Updated: 2025-11-05</p>
+<p class="doc-info">Version: 1.0.0</p>
+<p class="doc-info">Severity: Medium</p>
+<p class="doc-info">CVSS Score: 5.2 (CVSS:3.1/AV:A/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N)</p>
+<p class="doc-info">CVE IDs: <a href="https://www.cve.org/CVERecord?id=CVE-2025-10853">CVE-2025-10853</a></p>
+---
+
+### AFFECTED PRODUCTS
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)